loadouts/factions/structures/us_structure.sqf

```diff
@@ -0,0 +1,430 @@
+// INFO ========================================================================
```

Comment:
I feel that too, the only specific thing about the US one right now is the M320 launcher added to recon backpack TL, could change in a future update if you're ok with it, I still need to think about it, the German and French structure are also very similar and could be dissolved with the new changes since it only adds a couple of attachments to weapons.